wiki/docs/developer-resources/functional-spec.md

```diff
@@ -0,0 +1,218 @@
+2. *Microphone device*. Available options: HassMic (preferred), StreamAssist, ...
+3. *Media player device*. Available options: browsermod media player, HassMic media player, ...
+4. *Music player device*. Can be the same as the media player device, so same options are available. Additional options like Music Assistant are available (and most common).
+
+## Integration setup
+The VA integration can be found in the HA integration screen and will walk the user through a series of steps (config flow) to set up the integration and a satellite. The satellite is represented as a HA sensor with attributes. If the user wants to configure multiple satellites they simply start the config flow again. One set up of the first satellite the panel is created. We will have one panel, even if there are multiple satellites set up. 
+
+### Config flow
+These are the settings provided during config flow:
+1. **Type**: device with display or without
+2. **Name**: the unique name of the satellite
+3. **Microphone device**: the entity to use as the microphone device. This devices captures the intent of the user speaking to the satellite.
+4. **Media player device**: the entity to use as the mediaplayer for the satellite. TTS output will be sent to this media player device.
+5. **Music player device**: the entity to use as the musicplayer for the satellite. This can be different or the same as the media player device. Will be used to play music to.
+6. **Display device**: which entity to use to display satellite output on. Only available for satellites that are configured as type display. This should be set to the browser path entity of browser mod for the satellite.
+7. **Browser ID**: the ID of the browser mod installation. Only available for satellites that are configured as type display. 
+
+### Options flow
+The options flow is available after finishing the config flow for a satellite and will give the user access to the following settings so they can change these without having to delete and re-create their satellite config:
+1. Type
+2. Microphone device
+3. Media player device
+4. Music player device
+5. Display device
+6. Browser ID
+7. Specific weather entity to use (to override weather entity configured for all satellites)
+
+These settings are in the options flow because we don't feel like the user needs to change them that often after they are set up (principle ##5).
+
+## What the integration set up creates
+Succesfully completing at least one config flow of the integration creates the following items:
+- **a sensor** representing each satellite. Each satellite has its own sensor entity
+- **a panel** that hosts the views. There is only one panel and all satellites share the same panel but can have specific configurations for the views in the panel (see below).
+- **a group** hosting all the satellites. There is only one group of satellites.
+- **a timer** helper for each satellite. there is only one timer per satellite.
```

Comment:
I don't think this is possible, we will have to check with core team.

Timer isn't it's own platform in HA (unlike Sensor). But we might not need this if we can rely on events instead.

Reply:
This was meant for the group line not timer.  Apologies